src/client/datascience/interactive-window/nativeInteractiveWindow.ts

```diff
@@ -615,7 +626,10 @@ export class NativeInteractiveWindow implements IInteractiveWindowLoadable {
         const editor = window.visibleNotebookEditors.find((editor) => editor.document === this.notebookDocument);
         if (editor) {
             const notebookRange = new NotebookRange(notebookCell.index, notebookCell.index + 1);
-            editor.revealRange(notebookRange, NotebookEditorRevealType.InCenterIfOutsideViewport);
+            // This will always try to reveal the whole cell--input + output combined
+            setTimeout(() => {
+                editor.revealRange(notebookRange, NotebookEditorRevealType.Default);
+            }, 200); // Rendering output is async so the output is not guaranteed to immediately exist
```

Comment:
Would zero work as well here? Not sure about 200. Seems rather arbitrary.

Reply:
200 is more likely to work than 0, though note this doesn't work perfectly because outputs render asynchronously. For this to work properly we would need some kind of event to fire signaling that output has fully rendered

Reply:
> event to fire signaling that output has fully rendered

Rendered or finished executing?

Reply:
Rendered. We know when the cell has finished executing. The problem is that when we do `replaceOutputs`, the outputs are not guaranteed to have completely rendered when `replaceOutputs` resolves.